### PR TITLE
Todo List project

### PR DESCRIPTION
**Pull request by Emmanuelaaron**
### Tools and Techs used
- JavaScript
- HTML and CSS
- Bootstrap
- Eslint

### CODE REVIEW

File: src/functions/project.js [hunk 1]
```diff
@@ -0,0 +1,23 @@
+const projectsArray = [];
+
+function Project(name) {
```
Comment (whiz25) [inline src/functions/project.js]: - Use ES6 arrow function syntax

File: src/functions/project.js [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+  this.name = name;
+  this.todos = [{
+    name: 'running',
+    description: 'running is cool',
+    dueDate: '09/08/2019',
+    priority: 'high',
+  }];
+}
+
+function Todo(name, description, dueDate, priority) {
```
Comment (whiz25) [inline src/functions/project.js]: - Replace with ES6 arrow function syntax

File: package.json [hunk 3]
```diff
@@ -0,0 +1,55 @@
...
+  "version": "1.0.0",
+  "description": "",
+  "private": true,
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1",
+    "build": "webpack",
+    "watch": "webpack --watch",
+    "scss": "node-sass --watch scss -o css",
+    "start": "webpack serve"
+  },
```
Comment (whiz25) [inline package.json]: - Update the `scripts` with:
```json
"scripts": {
    "dev": "webpack --mode development",
    "build": "webpack --mode production",
    "start": "webpack-dev-server --mode development --open" // optional
  },

```

**Review by whiz25 — CHANGES_REQUESTED** [review]
## Changes Required ✅ ✅ ✅ 
Nice work on this project. You made quite progress.
### Highlights 👍🏿 👍🏿 👍🏿 
- A user can create a new project, todo item
- A user can edit a todo
- A user can delete a todo

### Changes to make 🌵 🌵 🌵 
- Make changes to the `package.json` file. Check my comments.
- Use ES6 syntax to declare functions. Check comments


### Martin Afani
#### Slack: whiz25

**Review by MuriloRoque — APPROVED** [review]
## Status: Approved :green_circle: 

### Hello @Emmanuelaaron :wave: 

### Thank you for making all of the requested changes. Your project is approved :clap: :clap: :clap: 

### You may merge, good luck with your next projects :rocket: 

Murilo Roque Paiva da Silva

- Github: [@muriloroque](https://github.com/MuriloRoque)
- Linkedin: [MuriloRoque](https://www.linkedin.com/in/murilo-roque-b1268741/)

[Twitter Follow](https://twitter.com/MuriloRoquePai1)

**Pull request merged**